packages/@lwc/engine-core/src/framework/vm.ts

```diff
@@ -439,15 +439,25 @@ function patchShadowRoot(vm: VM, newCh: VNodes) {
 }
 
 function runRenderedCallback(vm: VM) {
-    if (isTrue(vm.renderer.ssr)) {
+    const {
+        renderer,
+        def: { renderedCallback },
+    } = vm;
+
+    if (isTrue(renderer.ssr)) {
         return;
     }
 
     const { rendered } = Services;
     if (rendered) {
         invokeServiceHook(vm, rendered);
     }
-    invokeComponentRenderedCallback(vm);
+
+    if (!isUndefined(renderedCallback)) {
+        logOperationStart(OperationId.RenderedCallback, vm);
+        invokeComponentCallback(vm, renderedCallback);
+        logOperationEnd(OperationId.RenderedCallback, vm);
```

Comment:
tl;dr
This change will include the errorCallback time in renderedCallback.

If there was an error during renderedCallback, the `logOperationEnd` is invoked after invoking the error callback. Previously, it used to log the end and then invoke the error callback.

Reply:
Yes, this is a visible change. That said I think that it's for the best as it aligns with how the other lifecycle callbacks are invoked.